virttest/utils_disk.py

```diff
@@ -793,6 +793,25 @@ def configure_empty_disk(session, did, size, ostype, start="0M", n_partitions=1,
                                       labeltype, timeout)
 
 
+def check_disk_func_linux(session, did):
+    """
+    Check basic functions for linux guest disk.
```

Comment:
It seems the function can only support local guest session. So it is better to add 'local' in L798.

Reply:
updated.